domino/flyte/task.py

```diff
@@ -149,6 +170,7 @@ def _resolve_job_properties(self, domino_job_config: DominoJobConfig) -> Dict[st
             resolved_job_config["apiKey"] = domino_job_config.ApiKey
             resolved_job_config["command"] = domino_job_config.Command
             resolved_job_config["title"] = domino_job_config.Title
+            resolved_job_config["pipelineConfig"] = asdict(PipelineConfig())
```

Comment:
Since we just return a dict with no required keys and you're just inserting an empty `PipelineConfig` its probably fine to omit this line altogether.